pkg/parser/ast/functions.go

```diff
@@ -382,6 +382,12 @@ const (
 	SetVal  = "setval"
 )
 
+// IsVectorIndexDistanceMetricSupported stores distance metrics for the vector index.
+var IsVectorIndexDistanceMetricSupported = map[string]struct{}{
+	VecCosineDistance: struct{}{},
+	VecL2Distance:     struct{}{},
+}
+
```

Comment:
Unused variable?

Reply:
Yes, for the next pr, I deleted it first.